ext/libzip/zip_source_file_win32_utf16.c

```diff
@@ -95,7 +119,8 @@ utf16_create_file(const char *name, DWORD access, DWORD share_mode, PSECURITY_AT
 
     return CreateFile2((const wchar_t *)name, access, share_mode, creation_disposition, &extParams);
 #else
-    return CreateFile2FromAppW((const wchar_t *)name, access, share_mode, creation_disposition, NULL);
+    HANDLE h = CreateFile2FromAppW((const wchar_t *)name, access, share_mode, creation_disposition, NULL);
+	return h;
```

Comment:
This change doesn't look very necessary?  We should limit patches to ext/ code.

-[Unknown]